sympy/stats/crv_types.py

```diff
@@ -956,16 +956,8 @@ def Exponential(name, rate):
     >>> skewness(X)
     2
 
-    >>> X = Exponential('x', 10)
```

Comment:
why has this been removed?

Reply:
The idea was to show the precise working of the exponential distribution. Substituting a number in place of `lambda` was obvious and increased the length of docs unnecessarily. Examples are shown with the help of a general parameter `l`.
Complete properties of Exponential distribution are described in form of a general parameter `lambda`.
https://github.com/sympy/sympy/blob/f552493265c8f15695f8254eb1812873c25f33e3/sympy/stats/crv_types.py#L942-L960

Reply:
I don't see the point of removing documentation code. A few more examples are OK.